benchmark/reformat_input.py

```diff
@@ -337,19 +339,192 @@ def reformat_treepuzzle(gene_tree,
     msa_fa.write(output_msa_phy_fp, format='phylip')
 
 
+def _merge_genbank_seqs(genbank_fp):
+    """ Merge one to multiple sequences in a GenBank file into one.
+
+    Parameters
+    ----------
+    genbank_fp: string
+        file path to genome in GenBank format
+
+    Returns
+    -------
+    tuple of (
+        skbio.Sequence,
+            Genome sequence and metadata
+        dict of { list of [ string, int, int, string ] }
+            Gene name : translation, start, end, and strand
+    )
+    """
+    loci = []
```

Comment:
I think lines 359->371 can be merged. In the current code you open the file, count the number of sequences, close it, and then ask each of the sequences to Sequence.read. I think you could simply do: `for i, gb  in enumerate(Sequence.read(genbank_fp, format='genbank'))`. Obviously you could not use i, if not actually need it.